plugins/Lights/Lights.h

```diff
@@ -62,10 +63,12 @@ class Lights: public QObject
 
 private:
     light_device_t* m_lightDevice;
+    QDBusInterface* m_hfdInterface;
     QColor m_color;
     State m_state;
     int m_onMs;
     int m_offMs;
+    bool m_usesHfd;
```

Comment:
IIRC, POD type doesn't initialize automatically. Set the default value here, or in initializer list near the constructor, or the constructor itself.

Or, actually, consider eliminating this variable, and initialize `m_hfdInterface` in `init()` instead.